core/src/scss/components/lockup/_lockup--option-a.scss

```diff
@@ -33,7 +33,8 @@
 
     @include grid-media('md') {
       font-size: 3.2rem;
-      line-height: 0.7em;
+      line-height: 1.15em;
+      @include margin(-8px null -6px null);
```

Comment:
Hey so I know where this is coming from (a certain site that was launched this morning), is there a reason why they don't use option-m for this instead? Totally understand if it's a D8/Drupal thing or that option-m isn't available right now.

![Screen Shot 2020-08-24 at 5 03 11 PM](https://user-images.githubusercontent.com/42749717/91108096-dfbe6300-e62b-11ea-9447-a3a415b1f931.png)

Reply:
Great question. I'll ask.
https://stanfordwebservices.slack.com/archives/GJQJ7HTK7/p1598373210028400

Reply:
We're going to add lockup option M to the list in Drupal and adjust the default line height for option A in stanford_basic.

Reply:
> Great question. I'll ask.
> https://stanfordwebservices.slack.com/archives/GJQJ7HTK7/p1598373210028400

I can't seem to access this Slack channel. Let me know if I should be on it 😃

Reply:
You have been added.